app/controllers/catalog_controller.rb

```diff
@@ -245,6 +245,7 @@ class CatalogController < ApplicationController
     config.add_show_field "previous_frequency_tsim", label: "Previous Frequency", helper_method: :list
     config.add_show_field "index_finding_aid_note_tsim", label: "Index/Finding Aid Note", helper_method: :list
     config.add_show_field "awards_tsim", label: "Awards", helper_method: :unstyled_list
+    config.add_show_field "indigenous_subject_ssim", label: "First Nations (AIATSIS) Subject", helper_method: :subject_search_list
```

Comment:
So, is there a new Solr field with only AIATSIS subjects in it or are they indexed into the same `subject_ssim` field as non-AIATSIS subjects? 

If there is a new AIATSIS field in Solr, the `subject_search_list` helper will not work here, because the keyword values it adds to the search URL are coming from the `subject_ssim` field.

If there isn't a new AIATSIS field in Solr, the subject search will contain these unrelated subjectes. Is that what we really want?

Reply:
I see that there's a new AIATSIS specific field being indexed in this PR for catalogue-services: I see that there is a new field being indexed in https://github.com/nla/catalogue-services/pull/321

In this case, the `subject_search_list` will not work because it needs to pull the search values from `aiatsis_subject_ssim` instead.

Please create a new helper method to do this.